frontend/tests/features/wheresMyGene/landingPageFilter.test.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Test suite for select filter-related utils.
+ */
+import { expect, test } from "@playwright/test";
+import {
+  checkPlotSize,
+  checkSourceData,
+  deSelectFIlterOption,
+  goToWMG,
+  selectFIlterOption,
+  selectTissueandGeneOption,
+} from "../../utils/xgeneUtils";
+import { isDevStagingProd } from "tests/utils/helpers";
+const CHEVRON_LEFT = '[data-icon="chevron-left"]';
+
+const { describe, skip } = test;
+
+describe("Left side bar", () => {
+  skip(!isDevStagingProd, "WMG BE API does not work locally or in rdev");
+  test.beforeEach(async ({ page }) => {
+    // navigate to url
+    await goToWMG(page);
+    await selectTissueandGeneOption(page);
+  });
+
+  test("Left side bar collapse and expand", async ({ page }) => {
+    // click chevron left to collapse the left tab
+    await page.locator(CHEVRON_LEFT).click();
+
+    // verify the left tab is collapsed
+    expect(await page.getByTestId("add-organism").isVisible()).toBeFalsy();
+  });
+
+  test("Should be able select and de-select options for datasetfilter", async ({
+    page,
+  }) => {
+    const countBeforeFIlter = await checkSourceData(page);
```

Comment:
Seems like we might be able to extract more common code to keep these similar tests more DRY? Thank you!

Reply:
Hi @tihuan  please I would need more context here. Thanks!

Reply:
And @victorikeokoro6  pls add one liner comments to help future maintainers